release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -113,6 +113,11 @@ For more information, see xref:../security/audit-log-policy-config.adoc#configur
 [id="ocp-4-9-networking"]
 === Networking
 
+[id="ocp-4-9-ptp-fast-event-notifications"]
+==== Monitoring PTP fast events with the PTP fast events notifications framework
```

Comment:
I think the second occurrence of 'events' here should be singular. 
So:
...with the PTP fast event notifications framework